web/components/new-contract/contract-params-form.tsx

```diff
@@ -154,6 +155,19 @@ export function ContractParamsForm(props: {
     ? parseFloat(initialValueString)
     : undefined
 
+  const closeDateMap: { [key: string]: number | string } = {
+    'A day': 1,
+    'A week': 7,
+    '30 days': 30,
+    'This year': daysLeftInTheYear,
+  }
+
+  const NEVER_IN_DAYS = 30 * 12 * 1000 // ~1000 years
```

Comment:
I think Twitter polls always close (seemingly too soon — I feel like I often still want to vote in them).

I'm ok trying out this Never option. We'll see if users want it.